db/db_mysql/migrations/20160118194630_init.sql

```diff
@@ -0,0 +1,28 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+CREATE TABLE IF NOT EXISTS users (id integer primary key auto_increment,username varchar(255) NOT NULL UNIQUE,hash varchar(255),api_key varchar(255) NOT NULL UNIQUE );
+CREATE TABLE IF NOT EXISTS templates (id integer primary key auto_increment,user_id bigint,name varchar(255),subject varchar(255),text varchar(255),html varchar(255),modified_date datetime );
```

Comment:
The "html" and "text" fields should probably be a `BLOB` or `TEXT` - varchar is limiting in Mysql.